libraries/AP_RangeFinder/AP_RangeFinder_BLPing.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include "RangeFinder.h"
+#include "RangeFinder_Backend.h"
+
+class AP_RangeFinder_BLPing : public AP_RangeFinder_Backend
+{
+
+public:
+
+    // constructor
+    AP_RangeFinder_BLPing(RangeFinder::RangeFinder_State &_state,
+                            AP_RangeFinder_Params &_params,
```

Comment:
indent issue

Reply:
... may also be able to sneak some `const`s in...

Reply:
fixed indent and added const to params which we never actually use.